src/info/langs/language.tera

```diff
@@ -107,6 +107,14 @@ impl Language {
             {% endfor %}
         }
     }
+
+    pub fn get_language_icon(&self) -> &'static str {
```

Comment:
`Language::get_language_icon` is repetitive, and this can be `get_icon` or `get_nerd_font_icon`.

Reply:
is there any preference between `get_icon` and `get_nerd_font_icon`?

Reply:
Since this will mostly be used internationally, the shorter name is probably fine.